packages/lesswrong/lib/collections/comments/schema.ts

```diff
@@ -875,7 +875,42 @@ const schema: SchemaType<"Comments"> = {
     canRead: ['guests'],
     canUpdate: ['sunshineRegiment', 'admins'],
     canCreate: ['members', 'sunshineRegiment', 'admins'],
-  } 
+  },
+
+  doppelComments: {
+    // Resolved by doppelComments resolver
+    type: Array,
+    optional: true,
+    canRead: ['guests'],
+    canCreate: ['admins'],
+    canUpdate: ['admins'],
+    hidden: true,
+  },
+
+  'doppelComments.$': {
+    type: String,
+  },
+
+  ownDoppelCommentVote: resolverOnlyField({
+    type: 'DoppelCommentVote',
+    graphQLtype: 'DoppelCommentVote',
+    canRead: ['guests'],
+    resolver: async (comment: DbComment, args: void, context: ResolverContext) => {
+      const { DoppelCommentVotes } = context;
+      return await DoppelCommentVotes.findOne({commentId: comment._id, userId: context.currentUser?._id});
+    },
+  }),
+
+  doppelCommentVotes: resolverOnlyField({
+    type: 'DoppelCommentVote[]',
```

Comment:
I think this is not the right `type` to be passing into `resolverOnlyField`; it should be e.g. `Array` (typescript's Array constructor, not the string value `'Array'`)